alexa/skill/news.php

```diff
@@ -68,7 +68,7 @@ public function news_request( $request, $response ) {
 					$args = array(
 						'post_type' => voicewp_news_post_types(),
 						'posts_per_page' => 5,
-						'tax_query' => isset( $tax_query ) ? $tax_query : array(),
+						'tax_query' => isset( $tax_query ) ? array( $tax_query ) : array(),
```

Comment:
It looks like `$tax_query` is not instantiated here, which means it will always return an empty array.  Is this the desired functionality?